daemon/cluster/filters.go

```diff
@@ -99,7 +99,6 @@ func newListTasksFilters(filter filters.Args, transformFunc func(filters.Args) e
 
 func newListSecretsFilters(filter filters.Args) (*swarmapi.ListSecretsRequest_Filters, error) {
 	accepted := map[string]bool{
-		"names": true,
```

Comment:
We have to accept this still (for few version and for API versions 1.25 and 1.26). We need to "version" the accepted list of filter.